src/pngcodec.c

```diff
@@ -279,7 +279,7 @@ gdip_load_png_image_from_file_or_stream (FILE *fp, GetBytesDelegate getBytesFunc
 		png_set_read_fn (png_ptr, getBytesFunc, _gdip_png_stream_read_data);
 	}
 
-	png_read_png (png_ptr, info_ptr, 0, NULL);
+	png_read_png (png_ptr, info_ptr, PNG_TRANSFORM_STRIP_16, NULL);
```

Comment:
please add a short comment above with a short description of why we're doing this and e.g. mention it's a partial fix for bug XXX.

And also add the full description from the PR to the commit message.